ui/core/src/hooks/addLiquidityCalculator.ts

```diff
@@ -33,10 +33,37 @@ export function usePoolCalculator(input: {
   const tokenBField = useField(input.tokenBAmount, input.tokenBSymbol);
   const balanceMap = useBalances(input.balances);
 
+
+  const preExistingPool = computed(() => {
+    if (!tokenAField.asset.value || !tokenBField.asset.value) {
```

Comment:
It does not 100% mean that there is no existing pool, it could be the case that pool was created, but then one or both assets withdrawn completely. Most likely will never happen, but possible technically.

Reply:
let's do some testing after moving things around